test_ngi_config.yaml

```diff
@@ -0,0 +1,150 @@
+# An edited copy of this file should be placed under $HOME/.ngiconfig/ngi_config.yaml
+# or its path exported as the environment variable NGI_CONFIG
+
+analysis:
+    workflows:
+        NGI:
+            analysis_engine: ngi_pipeline.engines.piper_ngi
+        mate_pair:
+            analysis_engine: ngi_pipeline.engines.de_novo_pipeline
+        RNA:
+            analysis_engine: ngi_pipeline.engines.bcbio_ngi
+    best_practice_analysis:
+        whole_genome_reseq:
+            analysis_engine: ngi_pipeline.engines.piper_ngi
+        IGN:
+            analysis_engine: ngi_pipeline.engines.piper_ngi
+        qc:
+            analysis_engine: ngi_pipeline.engines.qc_ngi
+    # top_dir is the directory used by the pipeline as a starting point to build up the analysis run
+    # see ngi_pipeline/utils/filesystem.py for details.
+    top_dir: /proj/a2014205/nobackup/NGI/analysis_ready
+    sthlm_root: a2014205 
```

Comment:
if this is a test it would be better if we do not use production names, than the person going to deploy will adapt it to his/her needs